src/column_family.rs

```diff
@@ -48,3 +48,30 @@ pub struct ColumnFamily {
 }
 
 unsafe impl Send for ColumnFamily {}
+
+/// A specialized opaque type used to represent a column family. Used for multi-threaded
+/// mode. Clone (and Copy) is derived to behave like &ColumnFamily (used for
+/// single-threaded). Clone/Copy is safe because this lifetime is bound to DB like
```

Comment:
oh, maybe I need to implement `Send` as well?

Reply:
... and add a case for this there: https://github.com/rust-rocksdb/rust-rocksdb/pull/506/files#r602429497

Reply:
done